keps/wg-lts/20200122-kubernetes-yearly-support-period.md

```diff
@@ -0,0 +1,313 @@
+---
+title: Kubernetes Yearly Support Period
+authors:
+  - "@imkin"
+  - "@liggit"
+  - "@tpepper"
+  - "@jberkus"
+  - "@youngnick"
+owning-sig: wg-lts
```

Comment:
Should this be owned by @kubernetes/sig-release ?

Reply:
Yes

Reply:
Yep. Also, this should land in the SIG Release KEP subdirectory.

Reply:
Okay, I'll fix.